### PR TITLE
Tidy up btn rules

### PR DESCRIPTION
**Pull request by camillef**

### CODE REVIEW

File: assets/stylesheets/bootstrap/_buttons.scss [hunk 1]
```diff
@@ -67,21 +67,16 @@ a.btn {
...
 .btn-success,
 .btn-warning {
   @include button-variant-plain($btn-default-color, $btn-default-bg);
 }
-// Appears in Talis turqoise
-.btn-primary {
-  @include button-variant($btn-primary-color, $btn-primary-bg, $btn-primary-border);
-}
 
-// Info appears as pink
```
Comment (camillef) [inline assets/stylesheets/bootstrap/_buttons.scss:79]: Broken window: this comment is incorrect, and `btn-info` is actually exactly the same as `btn-default`, `btn-success` and `btn-warning` above

File: assets/stylesheets/bootstrap/_buttons.scss [hunk 2]
```diff
@@ -67,21 +67,16 @@ a.btn {
 // --------------------------------------------------
 
 .btn-default,
+.btn-info,
 .btn-success,
 .btn-warning {
   @include button-variant-plain($btn-default-color, $btn-default-bg);
 }
-// Appears in Talis turqoise
```
Comment (camillef) [inline assets/stylesheets/bootstrap/_buttons.scss:74]: Removing other comments before they also go out of date!!

File: assets/stylesheets/bootstrap/_buttons.scss [hunk 3]
```diff
@@ -98,7 +93,6 @@ a.btn {
 // Link buttons
 // -------------------------
 
-// Make a button look and behave like a link
```
Comment (camillef) [inline assets/stylesheets/bootstrap/_buttons.scss:101]: It doesn't really look and behave like a link any more...

File: assets/stylesheets/bootstrap/_variables.scss [hunk 4]
```diff
@@ -28,7 +28,7 @@ $gray:                   $gray700 !default;
 $gray-dark:              $gray800 !default;
 $gray-darker:            $gray900 !default;
 
-$turqoise: #017d87;
+$turquoise: #017d87;
```
Comment (camillef) [inline assets/stylesheets/bootstrap/_variables.scss:31]: Broken window: spelling!

File: assets/stylesheets/bootstrap/mixins/_buttons.scss [hunk 5]
```diff
@@ -72,24 +72,22 @@
   }
 }
 
-@mixin button-variant($color, $background, $border) {
```
Comment (camillef) [inline assets/stylesheets/bootstrap/mixins/_buttons.scss:75]: `$border` is always the same as `$background` for this button, so do as we did for `button-variant-plain` above and remove the extra param

**Review by danielmatthew — APPROVED** [review]

**Pull request merged**